src/js/utils/__tests__/FormUtil-test.js

```diff
@@ -21,21 +21,25 @@ describe('FormUtil', function () {
       let result = FormUtil.getMultipleFieldDefinition(
         'variable',
         1,
-        this.definition
+        this.definition,
+        null,
+        2
       );
 
-      expect(result[0].name).toEqual('variable[1].key');
-      expect(result[1].name).toEqual('variable[1].value');
+      expect(result[0].name).toEqual('variable[2].key');
+      expect(result[1].name).toEqual('variable[2].value');
     });
 
     it('does not modify the definition', function () {
       let result = FormUtil.getMultipleFieldDefinition(
         'variable',
         1,
-        this.definition
+        this.definition,
+        null,
```

Comment:
In `SchemaForm` we pass `undefined` to this method, whereas here we pass `null`. It looks like they both produce the same result, so I would prefer to be consistent (and `null` makes more sense than `undefined` to me)